src/cpp/desktop/CMakeLists.txt

```diff
@@ -36,9 +36,9 @@ if(NOT WIN32)
       endif()
          
       if(APPLE)
-         set(QT_CANDIDATES 5.12.5)
+         set(QT_CANDIDATES 5.12.5 5.12.1)
       else()
-         set(QT_CANDIDATES 5.12.5 5.10.1)
```

Comment:
Since the issue only exists on macOS, I think it'd make sense to scope the fallback to that OS.

Reply:
Oops, yes, that's what I meant to do. :-)